Cargo.toml

```diff
@@ -14,6 +14,7 @@ members = [
     "crates/formats",
     "crates/notary/client",
     "crates/notary/server",
+    "crates/notary/tests-integration",
```

Comment:
This was not added in https://github.com/tlsnotary/tlsn/pull/537

Reply:
Sure, I left out other stuffs I wanna fix on a different PR, but wanted to include the notary integration test for this PR since the original PR scope should pass this test

Reply:
Ah i guess i could've raised a small PR to add that integration test first haha, will remember that next time, but it's the only extra thing outside of this PR's original scope